designer/server/src/main/scala/pl/touk/nussknacker/ui/validation/NodeValidator.scala

```diff
@@ -71,7 +71,8 @@ class NodeValidator(modelData: ModelData, fragmentRepository: FragmentRepository
       variableTypes: Map[String, TypingResult]
   )(implicit metaData: MetaData): ValidationContext = {
     GlobalVariablesPreparer(modelData.modelDefinition.expressionConfig)
-      .validationContextWithLocalVariables(metaData, variableTypes)
+      .emptyLocalVariablesValidationContext(metaData)
```

Comment:
Nice suggestion, I changed it a bit to `prepareValidationContextWithGlobalVariablesOnly`, but agree that it is better to expose what is included in the result instead of exposing what is isn't.